features/22-email-verification/api.md

```diff
@@ -115,10 +172,10 @@ To check for verification:
 
 ```javascript
 // verify email 
-skygear.auth.verifyEmail('31DF2310FAA1')
-.then(() => {
+skygear.auth.verifyWithCode('email', '31DF2310FAA1')
```

Comment:
do we really need the field parameter (first parameter) in `verifyWithCode`? (A question, no idea)